db/seeds.rb

```diff
@@ -134,7 +120,7 @@
 DtodZip3Distance.import [:orig_zip3, :dest_zip3, :dist_mi], distances, { batch_size: 500, validate: false }
 
 puts 'Loading discounts from top TSP (by BVS) per channel...'
-if ENV['SEEDS_ENC_IV'].nil? || ENV['SEEDS_ENC_KEY'].nil?
+if ENV['SEEDS_ENC_IV'].blank? || ENV['SEEDS_ENC_KEY'].blank?
```

Comment:
If an environment variable is set like `export FOO=` (which we're doing in `bin/setup`), then `ENV['FOO']` will return an empty string, _not_ `nil`. This bit me locally. Luckily, `.blank?` will return `true` or `false` and considers both `nil` and an empty string to be `true`.

```
irb(main):001:0> ''.blank?
=> true
irb(main):002:0> nil.blank?
=> true
```